src/main.rs

```diff
@@ -0,0 +1,69 @@
+use clap::{Args, Parser, Subcommand};
+
+pub mod cfg;
+pub mod door;
+pub mod dos;
+
+//pub mod context;
+//pub mod doorsys;
+//pub mod write_dos;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
```

Comment:
You can just have a toplevel `enum Commands` instead, unless you're planning to add flags outside the subommands.